### ISSUE TITLE
[bug]: pf-button not focusable in Safari

### ISSUE THREAD

**Issue by Arathy-s** [patternfly/patternfly-elements#2569]
## Description of the issue

Keyboard navigation not working for pf-button in Safari.

### Impacted component(s)

pf-button and all the components built using pf-button.

### Steps to reproduce

Try opening https://patternflyelements.org/components/button/ in Safari and try to navigate to the button using the keyboard.
The button is not focusable.

**Note that this is not reproducable in GNOME web**, it affect Safari v16.5.2, perhaps other versions

### Expected behaviour

The button should be navigable using the keyboard as in Chrome, Firefox and Edge.

**Comment by bennypowers** [comment]
potential line of enquiry: is `delegatesFocus` not being applied to the subclass?

**Comment by brianferry** [comment]
Not sure if this is applicable, but it appears keyboard navigation has to be turned on in safari or at least it needed to be on my end https://www.tempertemper.net/blog/how-to-use-the-keyboard-to-navigate-on-safari.  After turning it on I was able to tab navigate through the buttons on https://patternflyelements.org/components/button/

**Comment by bennypowers** [comment]
why doesn't github have a "throwing the desk through the window" response emoji?

@Arathy-s does that solve your problem?

**Comment by Arathy-s** [comment]
Thank you @bennypowers and @brianferry, enabling the accessibility settings in the Safari browser solved the issue.
